dlt/dlt_broker.py

```diff
@@ -82,17 +83,33 @@ def __init__(
             self.filter_ack_queue = None
             self.filter_ack_msg_handler = None
 
-        kwargs["ip_address"] = ip_address
-        kwargs["port"] = port
-        kwargs["timeout"] = kwargs.get("timeout", DLT_CLIENT_TIMEOUT)
-        self.msg_handler = DLTMessageHandler(
-            self.filter_queue,
-            self.message_queue,
-            self.mp_stop_flag,
-            kwargs,
-            dlt_time_value=self._dlt_time_value,
-            filter_ack_queue=self.filter_ack_queue,
-        )
+        self._from_dlt_client = True if ip_address else False
```

Comment:
The ip_address is now an optional param in the __init__ signature, so it should have a default value of None

Reply:
Done, and docstring is updated accordingly